libr/core/cmd_info.c

```diff
@@ -1255,6 +1255,19 @@ static int cmd_info(void *data, const char *input) {
 					break;
 				}
 				bool fullGraph = true;
+				if (input[2] == ' ' && strcmp(input + 3, "")) {
+					const char *match = r_str_trim_head_ro (input + 3);
+					r_list_foreach (obj->classes, iter, cls) {
+					        if (cls->super && strstr (cls->super, match)) {
+							r_cons_printf ("agn %s\n", cls->super);
```

Comment:
this can introduce code injection vulnerabilities. quote the whole command and filter the cls->super value. maybe we can have a filtering api to do that automatically